src/Sylius/Bundle/FlowBundle/spec/Process/Context/ProcessContextSpec.php

```diff
@@ -46,7 +46,7 @@ function it_initializes(
     ) {
         $process->getScenarioAlias()->shouldBeCalled();
         $storage->initialize(Argument::type('string'))->shouldBeCalled();
-        $process->getOrderedSteps()->shouldBeCalled()->willReturn(array($previousStep, $currentStep, $nextStep));
+        $process->getOrderedSteps()->shouldBeCalled()->willReturn([$previousStep, $currentStep, $nextStep]);
```

Comment:
@stloyd :+1: 

@bendavies It's part of the [Command-Query Separation principle](http://martinfowler.com/bliki/CommandQuerySeparation.html).

Basically, methods should either:

  * Do something (_Command_, `shouldBeCalled()`)
  * Return something (_Query_, `willReturn()`)

but never both!

However, for methods that don't adhere to this principle, you'll need to do `->shouldBeCalled()->willReturn()`, but this is not the case here.